packages/puppeteer-extra-plugin-stealth/evasions/navigator.webdriver/index.js

```diff
@@ -15,11 +15,26 @@ class Plugin extends PuppeteerExtraPlugin {
     return 'stealth/evasions/navigator.webdriver'
   }
 
+  // Pre Chrome 88.0.4291.0
   async onPageCreated(page) {
     await page.evaluateOnNewDocument(() => {
       delete Object.getPrototypeOf(navigator).webdriver
     })
   }
+
+  // Post Chrome 88.0.4291.0
+  // Note: this will add an infobar to Chrome with a warning that an unsupported flag is set
+  // To remove this bar on Linux, run: mkdir -p /etc/opt/chrome/policies/managed && echo '{ "CommandLineFlagSecurityWarningsEnabled": false }' > /etc/opt/chrome/policies/managed/managed_policies.json
```

Comment:
Nice, but should we move this to the evasion's docs? Adding this to the jsdoc might help discovery. Also, according to these docs, the path for chromium is different:

https://chromium.googlesource.com/chromium/src.git/+/master/docs/enterprise/policies.md#policy-sources

```bash
#!/bin/bash
for PRODUCT_BASE in /etc/opt/chrome /etc/chromium; do
  POLICY_PATH=${PRODUCT_BASE}/policies/managed
  mkdir -p ${POLICY_PATH}
  echo '{"CommandLineFlagSecurityWarningsEnabled": false}' > ${POLICY_PATH}/managed_policies.json
done
```

wdyt?

Reply:
@jeroenvisser101 I thought about documenting it, it's a good point but I don't think it matters too much (it's more of a visual thing - unless it's possible to detect by checking the increased height of the top bar). Also the fix for Windows and macOS (if you want to document it properly) is a lot less trivial. But maybe you can make a quick PR?

The path being different for Chromium is a good one, we should probably link your shell snippet.

Reply:
Bynthe way, I wish there was a way to provide/overwrite the policies with a command line flag, but it looks like that is not possible.

Reply:
Would be good to have macOS/Windows examples as well

Reply:
I'm not actually sure how to set this up via macOS/Win, but am open to submit a PR for documentation + shell snippet. I will see if I can find time to check if the increased height is detectable. I think it is, but I want to confirm. I'm guessing this is only an issue for headful anyway. I think most run headless, but only because headful takes extra effort/resources to set up and run.

Reply:
For macOS it requires some fiddling with Profile Manager (some kind of enterprise-y product): https://sites.google.com/a/chromium.org/dev/administrators/mac-quick-start

For Windows it's even more ridiculous: https://sites.google.com/a/chromium.org/dev/administrators/windows-quick-start